packages/desktop-client/src/components/transactions/TransactionsTable.jsx

```diff
@@ -1349,10 +1349,13 @@ const Transaction = memo(function Transaction(props) {
   );
 });
 
-function TransactionError({ error, isDeposit, onAddSplit, style }) {
+function TransactionError({ error, isDeposit, onAddSplit, onDistributeRemainder, style, nestedTransactions }) {
   switch (error.type) {
     case 'SplitTransactionError':
       if (error.version === 1) {
+        const childTransactions = nestedTransactions.filter(t => t.is_child)
```

Comment:
We should just pass a `disabled` prop here and check for amounts of the calling component.

Reply:
The reason I didn't opt for this is that the calculation is only necessary if the case is `SplitTransactionError` and the error version is 1. If the calculation is done in the parent components, then it needs to be done for all kinds of errors. Would you still like me to move it?